frontend/src/App.jsx

```diff
@@ -19,7 +19,7 @@ function App() {
 					<Route path="/signup" element={<SignUpView />} />
 					<Route path="/notifications" element={<NotificationsPage />} />
 					<Route path="/messages" element={<MessagesPage />} />
-					<Route path="/profile/:username" element={<ProfilesPage />} />
+					<Route path="/profile/:id" element={<ProfilesPage />} />
```

Comment:
@Presson-coder I've been thinking about the way we're identifying users. What do you think is a better to identify users; by their usernames or their by their user ID? e.g /`profile/someuser` vs `profile/1234`

Reply:
I think the id's can work.  I was using the id because I noticed that the endpoint for posts doesn't contain the username field so I ended up using Id since it was there on both posts and profiles endpoints.

api/posts

[
    {
        "id": 1,
        "url": "http://chitchat-api.vndprojects.com/api/posts/1/",
        "user_profile": "http://chitchat-api.vndprojects.com/api/profiles/7/",
        "body": "Test post",
        "created_at": "2024-02-08T15:03:56.628379Z",
        "likes_count": 1
    },
    ....
    ]

Reply:
@Presson-coder We can change the API to return usernames instead of user ids. Let's keep using IDs for now, the API structure isn't set in stone, if you ever need information to be returned in a certain way that'll make it easier to consume in the frontend, let me know.

Reply:
@terrameijar Ok sure, I think that will be good. I'll let you know then, thank you.